.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.7'
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+      - name: Install GraphViz
+        run: |
+          sudo apt-get update
+          sudo apt-get install --no-install-recommends -y graphviz
+      - name: Install Ruby deps
+        run: |
+          echo 'gem: --no-document' >> ~/.gemrc
+          gem install asciidoctor pygments.rb && gem install --prerelease asciidoctor-tabs
+      - name: Install Docson
+        run: git clone -b v2.1.0 --depth 1 https://github.com/lbovet/docson.git
+      - name: Build
+        run: make
+      - name: Deploy
+        env:
+          BOT_SSH_KEY: ${{secrets.BOT_SSH_KEY}}
```

Comment:
@GreyCat I've just added `kaitai_struct_doc` to the list of selected repositories that can use `BOT_SSH_KEY` in GitHub Actions, so hopefully this migration should go smoothly:

![image](https://github.com/kaitai-io/kaitai_struct_doc/assets/47499687/0471c288-b051-4e40-877b-1c47329fd3bf)

Reply:
Sounds good! Thanks so much for this vital piece of infrastructure!